apps/ios/Shared/Views/UserSettings/SetDeliveryReceiptsView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SetDeliveryReceiptsView.swift
+//  SimpleX (iOS)
+//
+//  Created by Evgeny on 12/07/2023.
+//  Copyright © 2023 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+
+struct SetDeliveryReceiptsView: View {
+    @EnvironmentObject var m: ChatModel
+
+    var body: some View {
+        VStack(spacing: 16) {
+            Text("Delivery receipts!")
+                .font(.title)
+                .foregroundColor(.secondary)
+                .padding(.vertical)
+                .multilineTextAlignment(.center)
+
+            Spacer()
+
+            Button("Enable") {
+                m.setDeliveryReceipts = false
+            }
+            .font(.largeTitle)
+            Group {
+                if m.users.count > 1 {
+                    Text("Delivery receipts will be enabled for all contacts in all visible chat profiles.")
+                } else {
+                    Text("Delivery receipts will be enabled for all contacts.")
+                }
+            }
+            .multilineTextAlignment(.center)
+
+            Spacer()
+
+            Button("Enable later via Settings") {
```

Comment:
It could be same as for address, e.g.:

"Don't enable"

"You can change it via Settings"